tools/backup/templates/backup.yaml

```diff
@@ -57,17 +60,21 @@ spec:
                   value: "{{ .Values.checkLabelScanStore }}"
                 - name: CHECK_PROPERTY_OWNERS
                   value: "{{ .Values.checkPropertyOwners }}"
+              {{- if .Values.secretName }}
               volumeMounts:
```

Comment:
There is no way to mount a volume in the backup utility, if a user is trying to backup a big database and the local storage does not have enough space then the backup job will fail. We are currently facing this problem and to solve it we mounted a volume to the backup pod.